x/cellarfees/keeper/genesis.go

```diff
@@ -10,7 +12,19 @@ import (
 func InitGenesis(ctx sdk.Context, k Keeper, gs types.GenesisState) {
 	k.SetParams(ctx, gs.Params)
 
-	k.accountKeeper.GetModuleAccount(ctx, types.ModuleName)
+	feesAccount := k.GetFeesAccount(ctx)
+	if feesAccount == nil {
+		panic(fmt.Sprintf("%s module account has not been set", types.ModuleName))
+	}
+
+	// TODO(bolten): I added this check since other cosmos-sdk modules were doing this,
```

Comment:
I think a lot of the state machine coding can be a bit redundant and defensive like this. IDK if its necessary either. If this is how other modules work, might as well just leave it in tho.